projects/openshift.py

```diff
@@ -18,37 +18,92 @@
 # Refer to the README and COPYING files for full details of the license
 #
 
+import sys
+
 from fabric.api import cd
-from fabric.api import put
+from fabric.api import env
 from fabric.api import run
 from fabric.api import settings
 from fabric.api import sudo
 from fabric.api import task
-from fabric.api import env
-from fabric.contrib.files import sed
 
-from common import install_packages
 from common import enable_services
 from common import enable_tcp_ports
+from common import install_packages
+from manageiq import create_provider
+from manageiq import refresh_provider
 
 
+DEFAULT_ATTEMPTS = 10
+PROVIDER_NAME = 'openshift01'
+
 @task(name='install-deps')
 def install_deps():
     install_packages([
-        'docker-io',
+        'docker',
+        'docker-registry',
         'git',
         'golang',
         'screen',
         'vim',
     ])
+    # required on CentOS7
+    sudo('yum -y update device-mapper')
```

Comment:
Please open (and maybe fix) an issue on having a new `update_packages` method similar to `install_packages`.